qctrlopencontrols/dynamic_decoupling_sequences/driven_controls.py

```diff
@@ -212,45 +229,50 @@ def convert_dds_to_driven_control(
         detuning_rotations = np.append(detuning_rotations, [0])
 
     # check that the offsets are correctly sorted in time
-    if any(np.diff(offsets) <= 0.):
-        raise ArgumentsValueError("Pulse timing could not be properly deduced from "
-                                  "the sequence offsets. Make sure all offsets are "
-                                  "in increasing order.",
-                                  {'dynamic_decoupling_sequence': dynamic_decoupling_sequence},
-                                  extras={'offsets': offsets})
+    if any(np.diff(offsets) <= 0.0):
+        raise ArgumentsValueError(
+            "Pulse timing could not be properly deduced from "
+            "the sequence offsets. Make sure all offsets are "
+            "in increasing order.",
+            {"dynamic_decoupling_sequence": dynamic_decoupling_sequence},
+            extras={"offsets": offsets},
+        )
 
     offsets = offsets[np.newaxis, :]
     rabi_rotations = rabi_rotations[np.newaxis, :]
     azimuthal_angles = azimuthal_angles[np.newaxis, :]
     detuning_rotations = detuning_rotations[np.newaxis, :]
 
-    operations = np.concatenate((offsets, rabi_rotations,
-                                 azimuthal_angles, detuning_rotations),
-                                axis=0)
+    operations = np.concatenate(
+        (offsets, rabi_rotations, azimuthal_angles, detuning_rotations), axis=0
+    )
 
     pulse_mid_points = operations[0, :]
 
-    pulse_start_ends = np.zeros((
-        operations.shape[1], 2))   # pylint: disable=unsubscriptable-object
-
-    for op_idx in range(operations.shape[1]):   # pylint: disable=unsubscriptable-object
+    pulse_start_ends = np.zeros(
+        (operations.shape[1], 2)  # pylint: disable=unsubscriptable-object
```

Comment:
I think you can remove these, since the check is disabled in pylintrc (same with more of the same below).